spec/index.html

```diff
@@ -781,6 +781,37 @@ <h3>Collections</h3>
 
   </section>
 
+  <section id="triple-terms">
+    <h3>Triple Terms</h3>
+
+    <p>A <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>
+      may be the <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of an
+      <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>.</p>
+
+    <p>A <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>
+      is represented as a <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> with
+      <a href="#grammar-production-ttSubject"><code>ttSubject</code></a>,
+      <a href="#grammar-production-predicate"><code>predicate</code></a>, and
+      <a href="#grammar-production-ttObject"><code>ttObject</code></a>
+      preceded by <a href="#cp-double-lt-paren"><code>&lt;&lt;(</code></a>, and
+      followed by <a href="#cp-paren-double-gt"><code>)&gt;&gt;</code></a>.
+      Note that <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>
+      may be nested.
+     </p>
+
+    <pre id="ex-triple-term" class="example turtle" data-transform="updateExample"
+         title="Triple Term">
+      <!--
+      PREFIX :    <http://www.example.org/>
+      PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+      _:e38  :familyName                     "Smith" .
+      _:anno rdf:reifies <<( _:e38 :jobTitle "Designer" )>> .
+      _:anno :accordingTo                     _:e22 .
+      -->
```

Comment:
Being within a `<pre>`, these lines should not be indented.
```suggestion
<!--
PREFIX :    <http://www.example.org/>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>

_:e38  :familyName                     "Smith" .
_:anno rdf:reifies <<( _:e38 :jobTitle "Designer" )>> .
_:anno :accordingTo                     _:e22 .
-->
```

Reply:
Respec already unindents text block within a `pre` element. It's not necessary to left justify blocks, and interrupts the flow of the document.

Reply:
OK. ReSpec features are not yet all built in to my brain.